src/ngraph/op/experimental/quantized_conv.hpp

```diff
@@ -33,7 +33,9 @@ namespace ngraph
                                  const CoordinateDiff& padding_below,
                                  const CoordinateDiff& padding_above,
                                  const Strides& data_dilation_strides,
-                                 const std::shared_ptr<Node> scale);
+                                 const std::shared_ptr<Node> scale,
+                                 const std::shared_ptr<Node> input_scale = nullptr,
```

Comment:
Does this really work when `input_scale == nullptr`? In the ctor implementation over in `quantized_conv.cpp` you are going directly to calling `check_single_output_args` on this, and that function does not contain any `nullptr` checks.